src/sparseml/pytorch/torchvision/train.py

```diff
@@ -189,19 +200,26 @@ def evaluate(
                 output = output[0]
             loss = criterion(output, target)
 
-            acc1, num_correct_1, acc5, num_correct_5 = utils.accuracy(
-                output, target, topk=(1, 5)
-            )
```

Comment:
Thoughts on just making this `topk=(1, num_classes)`? Then we don't have to handle it being optional